packages/desktop-client/src/components/NotesButton.tsx

```diff
@@ -43,23 +43,23 @@ const markdownStyles = css({
   },
   '& blockquote': {
     paddingLeft: '0.75rem',
-    borderLeft: '3px solid ' + colors.p6,
+    borderLeft: '3px solid ' + theme.formInputBorderSelected,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏(this comment applies to the entire file)
I think we should create unique color variables for the markdown block. Since these are contextually different things than borders, etc.